index.js

```diff
@@ -40,14 +40,21 @@ function exchangeList () {
   return fetchJSON(url)
 }
 
+function constituentExchangeList (options) {
+  options = options || {}
+  let url = `${baseUrl}all/includedexchanges`
+  if (options.instrument) url += `?instrument=${options.instrument}`
+  return fetchJSON(url).then(result => result.Data)
+}
+
 function newsFeedsAndCategories () {
-  const url = `${baseUrl}data/news/feedsandcategories`
+  const url = `${baseUrl}news/feedsandcategories`
```

Comment:
Why are these changes still included?

Reply:
My forked repository has a previous version of master. I'll look into how to fetch the latest version of master (since I'm not sure how to besides `git fetch upstream`, which kept throwing an error) and rebase accordingly.